examples/kubernetes/konnectivity-agent.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: konnectivity-agent
+  namespace: default
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ''
+    seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
+spec:
+  hostNetwork: true
+  containers:
+  - name: konnectivity-agent-container
+    image: ${AGENT_IMAGE}:${TAG}
+    resources:
+      requests:
+        cpu: 40m
+    command: [ "/proxy-agent"]
+    args: [
+      "--logtostderr=true",
+      "--ca-cert=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+      "--proxy-server-host=${CLUSTER_IP}",
+      "--proxy-server-port=8091",
+      ]
+    livenessProbe:
+      httpGet:
+        scheme: HTTP
+        host: 127.0.0.1
+        port: 8093
+        path: /healthz
+      initialDelaySeconds: 15
+      timeoutSeconds: 15
+    resources:
+      limits:
+        cpu: 50m
```

Comment:
is this accurate?

Reply:
I assume so, it's a copy and paste of what we have in k/k (https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/addons/konnectivity-agent/daemonset.yaml#L53). cc: @cheftako

Reply:
A very good question. Its sufficient for the server to come up. It will be interesting to see if its sufficient to proxy something like a cluster with high admission webhook load.